synthtool/gcp/common.py

```diff
@@ -358,3 +371,8 @@ def _load_repo_metadata(metadata_file: str = "./.repo-metadata.json") -> Dict:
         with open(metadata_file) as f:
             return json.load(f)
     return {}
+
+
+def _get_help(filename: str) -> str:
+    """Function used by sample readmegen"""
```

Comment:
Please describe the return value in the doc comment.

Reply:
I'd rather see this one line of code be a lambda on line 63.  Is there an advantage of breaking it out into a separate function?